### PR TITLE
Huongpham1603/sc 32359/core be dtgk change config bnpl with the new flow

### PR DESCRIPTION
**Pull request by HuongPhamCredify**
## description
[Core] BE DTGK change config BNPL with the new flow

## shortcut
https://app.shortcut.com/credify/story/32359/core-be-dtgk-change-config-bnpl-with-the-new-flow?team_id=1&iteration_ids=32042

### CODE REVIEW

File: v1.js [hunk 1]
```diff
@@ -68,34 +68,38 @@ module.exports = () => {
      * }
      */
     const paymentRecipient = {
-      name: "Demo store",
-      number: "xxxxxxxxxxx",
-      branch: "",
-      bank: "XXX Bank",
-      type: "BANK_ACCOUNT"
-    }
+      name: "Công ty TNHH GIÁ KHO GROUP",
```
Comment (ngo275) [inline v1.js:71]: Please keep the original values because this is skeleton.

Comment (ngo275) [reply v1.js:71]: You need to add this level of details in the forked repos

File: v1.js [hunk 2]
```diff
@@ -68,34 +68,38 @@ module.exports = () => {
...
-      res.status(400).json({ error: { message: e.message } })
+      res.status(400).json({ error: { message: e.message } });
     }
-  })
+  });
 
-  api.post("/simulate", async (req, res) => {
-    const productType = "consumer-financing:unsecured-loan:bnpl"
+  api.post("/simulation", async (req, res) => {
+    const productType = req.body.product_type;
```
Comment (ngo275) [inline v1.js:95]: This is only for BNPL, so please keep this hardcoded value

File: v1.js [hunk 3]
```diff
@@ -68,34 +68,38 @@ module.exports = () => {
...
 
     // TODO: Please update this request body
 
     // This should be string array (can be empty)
-    const providerIds = req.body.provider_ids
+    const providerIds = req.body.provider_ids || [
+      "09de7359-7f29-41a0-bd07-095d1ce7f85d",
```
Comment (ngo275) [inline v1.js:101]: Please add this detail in the forked repo

Comment (HuongPhamCredify) [reply v1.js]: I fixed it

**Review by ngo275 — APPROVED** [review]

**Pull request merged**